.github/workflows/octokit.yml

```diff
@@ -43,7 +43,9 @@ jobs:
         run: chmod 600 spec/fixtures/.netrc
       - name: Install dependencies
         run: |
-          bundle install --path=.bundle/gems --without=development --jobs 4 --retry 3
+          bundle config set path .bundle/gems
+          bundle config set without development
+          bundle install --jobs 4 --retry 3
```

Comment:
Fix deprecation warnings for Bundler 3.
See <https://github.com/rubygems/bundler/blob/35be6d9a603084f719fec4f4028c18860def07f6/UPGRADING.md#cli-deprecations>.